src/docs/sphinx/advancedExamples/validationStudies/mccWellbore/Example.rst

```diff
@@ -0,0 +1,235 @@
+
+
+
+In this example, we focus our attention on the ``Mesh`` tags,
+the ``Constitutive`` tags, and the ``FieldSpecifications`` tags.
+
+------------------------------------------------------------------
+Mesh
+------------------------------------------------------------------
+
+Following figure shows the generated mesh that is used for solving this wellbore problem.
+
+
+.. _problemSketchFig:
+.. figure:: MCC_wellMesh.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Generated mesh for the wellbore problem
+
+
+Let us take a closer look at the geometry of this wellbore problem.
+We use the internal mesh generator ``InternalWellbore`` to create a rock domain
+(:math:`10\, m \, \times 5 \,  m \, \times 2 \, m`), with a wellbore of
+initial radius equal to :math:`0.1` m.
+Coordinates of ``trajectory`` defines the wellbore trajectory, which represents a vertical well in this example. 
+By turning on ``autoSpaceRadialElems="{ 1 }"``, the internal mesh generator automatically sets number and spacing of elements in the radial direction, which overrides the values of ``nr``. 
+With ``useCartesianOuterBoundary="0"``, a Cartesian aligned outer boundary on the outer block is enforced.
+In this way, a structured three-dimensional mesh is created with 50 x 40 x 2 elements in the radial, tangential and z directions, respectively. All the elements are eight-node hexahedral elements (``C3D8``) and refinement is performed
+to conform with the wellbore geometry. This mesh is defined as a cell block with the name
+``cb1``.
+
+
+.. literalinclude:: ../../../../../../src/coreComponents/physicsSolvers/solidMechanics/benchmarks/PlasticWell_Base.xml
+    :language: xml
+    :start-after: <!-- SPHINX_WELLBORE_MESH -->
+    :end-before: <!-- SPHINX_WELLBORE_MESH_END -->
+
+
+------------------------
+Solid mechanics solver
+------------------------
+
+For the drained wellbore problem, the pore pressure variation is omitted. Therefore, we just need to define a solid mechanics solver, which is called ``mechanicsSolver``. 
+This solid mechanics solver (see :ref:`SolidMechanicsLagrangianFEM`) is based on the Lagrangian finite element formulation. 
+The problem is run as ``QuasiStatic`` without considering inertial effects. 
+The computational domain is discretized by ``FE1``, which is defined in the ``NumericalMethods`` section. 
+The material is named as ``rock``, whose mechanical properties are specified in the ``Constitutive`` section.
+
+.. literalinclude:: ../../../../../../src/coreComponents/physicsSolvers/solidMechanics/benchmarks/PlasticWell_Base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_WELLBORE_SOLVER -->
+  :end-before: <!-- SPHINX_WELLBORE_SOLVER_END -->
+
+
+------------------------------
+Constitutive laws
+------------------------------
+
+For this drained wellbore problem, we simulate the elastoplastic deformation caused by wellbore expansion.
```

Comment:
```suggestion
For this undrained wellbore problem, we simulate the elastoplastic deformation caused by wellbore expansion.
```

Reply:
see above comment about the drained condition.